compiler/test/compilable/imports/aliasdump.d

```diff
@@ -0,0 +1,3 @@
+alias twinkleTwinkleKillerKane = ulong;  ///
```

Comment:
You can think of better names than that. Even `a23369` as a reference to the issue is fine.